plugins/fpc/fu-fpc-device.c

```diff
@@ -429,7 +583,16 @@ fu_fpc_device_write_firmware(FuDevice *device,
 	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 95, NULL);
 	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_BUSY, 5, "check");
```

Comment:
These progress may move for old format only, otherwise it would cause progress error because of not be done in new format.

Reply:
I thought I squashed them together -- my mistake. I'll fix up now.